aten/src/ATen/native/TensorIterator.h

```diff
@@ -85,7 +86,21 @@ struct CAFFE2_API OperandInfo {
   /// input should be converted to this type if necessary. For outputs, this
   /// specifies which type to allocate. Note that there is very limited support
   /// for type conversions currently: they are only allowed for zero-dim tensors.
-  Type* type = nullptr;
+  Backend backend = Backend::Undefined;
+  ScalarType dtype = ScalarType::Undefined;
+
```

Comment:
Docs for all of these? It's fairly clear what these do, but what I want to know is *when* I should use them

Reply:
Ah, reading through, I see these are just adjustments for methods you were previously calling on Type